File: arrays 101/height_checker.py
class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        count = 0
        l1 = []
        if len(heights) == 0:
            return 0
        l1 = sorted(heights)
        for i in range(len(heights)):
            if heights[i] != l1[i]:
                count += 1
        return count

File: arrays 101/height_checker.java
class Solution {
    public int heightChecker(int[] heights) {
        int count=0;
		
		int[] arr = new int[heights.length];
        
        if(heights.length==0) return 0;
		for(int i=0; i<heights.length; i++)
		{
			arr[i] = heights[i];
		}
		
		Arrays.sort(heights);
		
		for(int i=0; i<heights.length; i++)
		{
			if(arr[i] != heights[i])
			{
				count++;
			}
		}
		
        
        return count;
    }
}